oml/lightning/callbacks/metric.py

```diff
@@ -73,7 +73,9 @@ def on_validation_epoch_end(self, trainer: pl.Trainer, pl_module: pl.LightningMo
 
     def calc_and_log_metrics(self, pl_module: pl.LightningModule) -> None:
         metrics = self.metric.compute_metrics()
-        metrics = flatten_dict(metrics, sep="/")
+        # In Tensorboard/Neptune we only log metrics for the main category
+        metrics = {self.metric.overall_categories_key: metrics[self.metric.overall_categories_key]}
```

Comment:
pray remove it 
Or modify init in callback for this option

Reply:
ok